www-servers/lighttpd/lighttpd-1.4.64-r1.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} )
+
+inherit autotools flag-o-matic lua-single readme.gentoo-r1 systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Lightweight high-performance web server"
+HOMEPAGE="https://www.lighttpd.net https://github.com/lighttpd"
+SRC_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/${P}.tar.xz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm ~arm64 ~hppa ~ia64 ~mips ppc ppc64 ~s390 sparc x86"
```

Comment:
sorry, I'd missed this earlier, but on bumps, do `ekeyword \~all foo.ebuild` where `foo.ebuild` is the new version. See [here](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades).

Reply:
ackn